manta-pay/src/signer/client/network.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019-2022 Manta Network.
+// This file is part of manta-signer.
+//
+// manta-signer is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// manta-signer is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with manta-signer. If not, see <http://www.gnu.org/licenses/>.
+
+//! Manta Signer Multi-Network Support
+
+use core::{
+    fmt::{self, Display},
+    ops::{Index, IndexMut},
+};
+use manta_util::serde::{Deserialize, Serialize};
```

Comment:
Since `serde` does not exist by default you have to use the feature to write this.

```suggestion

#[cfg(feature = "serde")]
use manta_util::serde::{Deserialize, Serialize};
```